### ISSUE TITLE
Feature Request: Support Attachment

### ISSUE THREAD

**Issue by charno** [dmfs/opentasks#57]
I would like to see that attachments (at least inline) are supported in Tasks. A spec for CalDAV can be found at http://tools.ietf.org/html/rfc4791#section-8.5

My reason for this is that with Kontact I can automatically generate tasks out of Mails, and the mail gets attached to the task. On the other hand it would be for example handy to be able to attach images on the fly for tasks

**Comment by SilKaw** [comment]
Hello, how is the progress in adding the feature of attaching files (for example images) to tasks?

**Comment by Zvezdin** [comment]
Agreed. Plus the only way to attach links through KOrganizer or Thunderbird is via attachments, which are invisible to the app and as I see from separate issues, will get deleted once OpenTasks modifies the entry. 

Also, is the URL field of tasks standard-compliant? I can't see it from any desktop frontend.
